nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -1282,7 +1282,7 @@ def clone_without_ingress(config):
             return config  # no ingress policies in this config
         config_without_ingress = config.clone_without_policies(config.name)
         for policy in config.policies_container.policies.values():
-            if not isinstance(policy, IngressPolicy):  # ignoring ingress policies
+            if not isinstance(policy, GatewayPolicy):  # ignoring ingress policies
```

Comment:
consider renaming the function `clone_without_ingress`, as now the layer and resources are not named `ingress` but rather `gateway`

Reply:
Done